plugins/ping.rb

```diff
@@ -23,9 +32,39 @@ def match?(cmd)
     cmd =~ /^(!)?ping$/
   end
 
-  match /ping$/, method: :ping
+  def format_ping_time
+    milis = (Time.now - @timestamp) * 1000.0
+    mins = (milis / 60000).to_i
+    secs = (milis / 1000).to_i
+    str = ""
+    if mins > 0
+      str = "#{str}#{mins}"
+      str = "#{str}#{mins == 1 ? " minute, " : " minutes, "}"
```

Comment:
Instead of a ternary inside of these interpolations, let's just use `min` and `sec` as generic abbreviations for `minute(s)` and `second(s)`.

Reply:
should I still assume milliseconds will always be millisecond**s** or do millisecond(s) also?

Reply:
Do `millisecond(s)` as well.